resources/views/rss.blade.php

```diff
@@ -18,8 +18,8 @@
                 <author><![CDATA[{{ $item->author }}]]></author>
                 <guid>{{ url($item->id) }}</guid>
                 <pubDate>{{ $item->updated->toRssString() }}</pubDate>
-                @if($item->__isset('imageUrl') && $item->__isset('imageSize'))
-                    <enclosure url="{{ $item->imageUrl }}"  type="image/*" length="{{ $item->imageSize }}" />
+                @if($item->__isset('attachmentUrl') && $item->__isset('attachmentSize') && $item->__isset('attachmentType'))
```

Comment:
Shouldn't the `__isset()` method call be replaceable by a regular `isset()` function call? I think that would be easier to read/more common.

Reply:
I used this because this is a function in the FeedItem class: https://github.com/spatie/laravel-feed/blob/master/src/FeedItem.php#L149

Reply:
This is a PHP magic method which is triggered by calling `isset()` on an inaccessible property.
```php
isset($item->imageUrl)
```
https://www.php.net/manual/en/language.oop5.overloading.php#object.isset

Reply:
I removed the call to the magic method and used isset().
I also indented some lines for readability

Reply:
PS: I think that you should configure your local git or GitHub to use the same email(s). 😉
Right now your commits are unassigned. 😞